README.md

```diff
@@ -223,7 +223,7 @@ UseCase: Need to confirm a dynamically generated password
 Prerequisites: `docker`
 
 ```
-curl -s https://raw.githubusercontent.com/aws-quickstart/taskcat/master/installer/docker-install-master| sudo python -E
+docker pull taskcat/taskcat
```

Comment:
docker pull will just pull the container. It does not set up the mount point. 
So you need to run the installer! 
just back out this line and rest looks good to me